client/my-sites/posts/post-share.jsx

```diff
@@ -127,19 +128,34 @@ const PostSharing = React.createClass( {
 				{ this.props.failure && <Notice status="is-error" onDismissClick={ this.dismiss }>{ this.translate( `Something went wrong. Please don't be mad.` ) }</Notice> }
 				<div className={ classes }>
 					{ this.props.siteId && <QueryPostTypes siteId={ this.props.siteId } /> }
-					<h3 className="posts__post-share-title">
-						{ this.translate( 'Share the post and spread the word!' ) }
-					</h3>
-					<div className="posts__post-share-services">
-						{ this.renderServices() }
+					<h4 className="posts__post-share-title">
+						{ this.translate( 'Publicize your content' ) }
+					</h4>
+					<div className="posts__post-share-subtitle">
+						{ this.translate( 'Share your post on all of your connected social media accounts using {{a}}Publicize{{/a}}', {
+							components: {
+								a: <a href={ '/sharing/' + this.props.siteId } />
+							}
+						} ) }
 					</div>
-					{ this.renderMessage() }
-					<Button
-						onClick={ () => this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message ) }
-						disabled={ this.props.requesting || ( ( this.props.connections.length || 0 ) - this.state.skipped.length  < 1 ) }
-					>
-						{ this.translate( 'Share post' ) }
-					</Button>
+					{ this.hasConnections() && <div>
+							<div className="posts__post-share-services">
+								{ this.renderServices() }
+							</div>
+							{ this.renderMessage() }
+							<Button
+								onClick={ () => this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message ) }
```

Comment:
This will set off another `JSX props should not use arrow functions` warning. Mind moving this?

```javascript
sharePost() {
    this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message );
}
//...
<Button onClick={ this.sharePost } ... 
```